File: cfgs/java_cfgs_foo/FIND_MINIMUM_SUM_FACTORS_NUMBER.java
Nodes
1;special;;;
2;statement;int sum = 0;14;
3;statement;int i = 2;15;
4;conditional;(foo_1 * foo_2) <= foo_3;16;
5;conditional;(foo_1 % foo_2) == 0;18;
6;statement;sum += foo_1;19;
7;statement;num /= foo_1;20;
8;statement;i++;17;
9;statement;sum += foo_1;23;
10;statement;return foo_1;24;
11;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;9;
4;8;6;
5;7;
6;5;
7;4;
8;10;
9;11;
10;

File: cfgs/pythoncfgs/FIND_MINIMUM_SUM_FACTORS_NUMBER.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = 2;9;
3;conditional;foo_1 * foo_2 <= foo_3;10;
4;statement;foo_1 += foo_2;15;
5;statement;return foo_1;16;
6;conditional;foo_1 % foo_2 == 0;11;
7;statement;foo_1 += 1;14;
8;statement;foo_1 += foo_2;12;
9;statement;foo_1 /= foo_2;13;
10;exit;;
Edges
0;2;
1;3;
2;4;6;
3;5;
4;10;
5;7;8;
6;3;
7;9;
8;6;
9;
